package.json

```diff
@@ -43,7 +43,7 @@
     "icons"
   ],
   "peerDependencies": {
-    "react": "^15.0.0 || ^16.0.0 || ^17.0.0"
+    "react": ">=15.0.0"
```

Comment:
You should actually just add 18 to the list, cause otherwise you will automatically support 19 as well when it comes out, while it might actually be broken.

Reply:
Allright!!